### PR TITLE
Feat/add accessibilty plugin

### LINKED ISSUE #47: Add Nightwatch plugins to create Nightwatch
We need to add questions to ask people if they want to install

* Nightwatch VRT
* Nightwatch AXE
* Nightwatch Slack reporter

We will also need to have documentation that links to each of those items.

### PR DESCRIPTION
**Pull request by gravityvi**
## changes
- added Nightwatch-axe-plugin by default
- added example for accessibility tests

## Impacts
- fixes #47

### CODE REVIEW

**Review by AutomatedTester — APPROVED** [review]

File: src/init.ts [hunk 1]
```diff
@@ -378,7 +383,7 @@ export class NightwatchInit {
     const tplData = fs.readFileSync(templateFile).toString();
 
     let rendered = ejs.render(tplData, {
-      plugins: false,
+      plugins: JSON.stringify(plugins).replace(/"/g, '\'').replace(/\\\\/g, '/'),
```
Comment (garg3133) [inline src/init.ts:386]:
```suggestion
      plugins: JSON.stringify(plugins).replace(/"/g, '\''),
```
as plugins don't have path (this was added to replace `\` in windows path to `/`)

**Review by garg3133 — COMMENTED** [review]
We should also modify [here](https://github.com/nightwatchjs/create-nightwatch/blob/b28f63e0ad78edde558049b1149c92025a296c5c/src/config/main.ejs#L28-L32) to:
```js
// See https://nightwatchjs.org/guide/extending-nightwatch/adding-plugins.html
plugins: <%- plugins %>

// See https://nightwatchjs.org/guide/concepts/test-globals.html
```
as `plugins` now will always be truthy.

**Comment by gravityvi** [comment]
> We should also modify [here](https://github.com/nightwatchjs/create-nightwatch/blob/b28f63e0ad78edde558049b1149c92025a296c5c/src/config/main.ejs#L28-L32) to:
> 
> ```js
> // See https://nightwatchjs.org/guide/extending-nightwatch/adding-plugins.html
> plugins: <%- plugins %>
> 
> // See https://nightwatchjs.org/guide/concepts/test-globals.html
> ```
> 
> as `plugins` now will always be truthy.

Let's keep this. It is future proof.

File: tests/e2e_tests/init.js [hunk 2]
```diff
@@ -1119,19 +1124,21 @@ describe('e2e tests for init', () => {
...
+      assert.strictEqual(commandsExecuted[5], 'sudo safaridriver --enable');
     } else {
-      assert.strictEqual(commandsExecuted.length, 4);
+      assert.strictEqual(commandsExecuted.length, 5);
     }
     assert.strictEqual(commandsExecuted[0], 'npm install nightwatch --save-dev');
-    assert.strictEqual(commandsExecuted[1], 'npm install @nightwatch/selenium-server --save-dev');
-    assert.strictEqual(commandsExecuted[2], 'java -version');
-    assert.strictEqual(commandsExecuted[3], 'npm install chromedriver --save-dev');
+    assert.strictEqual(commandsExecuted[1], 'npm install nightwatch-axe-verbose --save-dev');
```
Comment (beatfactor) [inline tests/e2e_tests/init.js:1133]: I suggest we move this as a dependency on nightwatch itself.

**Comment by gravityvi** [comment]
Added `nightwatch-axe-verbose` package in `nightwatch` and the commands are loaded into client API by default. PR: https://github.com/nightwatchjs/nightwatch/pull/3389

**Pull request merged**